src/component.js

```diff
@@ -48,7 +48,7 @@ extend(Component.prototype, {
 
 	/** Update component state by copying properties from `state` to `this.state`.
 	 *	@param {object} state		A hash of state properties to update with new values
-	 *  	@param {function} callback	A function to be called once component state is updated
+	 *	@param {function} callback	A function to be called once component state is updated
```

Comment:
This probably was my editor behaving a bit funky, I didn't reconfigure it to work with preact codebase, sorry :)